lib/ImageHelper.php

```diff
@@ -328,7 +328,7 @@ protected static function process_delete_generated_files( $filename, $ext, $dir,
 	 */
 	public static function get_server_location( $url ) {
 		// if we're already an absolute dir, just return
-		if ( 0 === strpos($url, ABSPATH) ) {
+		if ( 0 === strpos($url, chr(ABSPATH)) ) {
```

Comment:
I don't think this `chr()` should be here either?

Reply:
Saw that too - pushed fix already. Thanks.